integration/nonmem.go

```diff
@@ -50,6 +50,19 @@ func AssertNonMemCreatedOutputFiles(t *wrapt.T, details NonMemTestingDetails) {
 	}
 }
 
+func AssertNonMemCleanedUpFiles(t *wrapt.T, details NonMemTestingDetails) {
+	t.Helper()
+
+	expectMissing := []string{
```

Comment:
I guess maybe consider whether we want to check for any other cleaned up files in this function? I don't have a strong feeling about that, but it does seem a little arbitrary to to just check these two. 

On the other hand, if you think it doesn't add much value and might get brittle to add more then I'm fine with leaving it.

Reply:
> it does seem a little arbitrary to to just check these two.

Fair enough.  I've pushed a change that now checks all of the level 1 files returned by `EstOutputFileCleanLevels`.

Locally I verified that it still fails when the relevant part of c062ae94 (nonmem: clean up FDATA.csv) is reverted.

<details>
<summary>regression check</summary>

```
ip-10-128-42-167:bbi$ ( make && cd integration && go test -run TestBbiCompletesLocalExecution )
cd cmd/bbi; go install -ldflags "-X main.buildTime=`date +%FT%T%z`"
PASS
ok      github.com/metrumresearchgroup/bbi/integration  6.583s

ip-10-128-42-167:bbi$ git diff
diff --git a/cmd/nonmem.go b/cmd/nonmem.go
index 43294bd..07082b7 100644
--- a/cmd/nonmem.go
+++ b/cmd/nonmem.go
@@ -76,7 +76,6 @@ var nonMemTemporaryFiles []string = []string{
        "compile.lnk",
        "FCON",
        "FDATA",
-       "FDATA.csv",
        "FMSG",
        "FREPORT",
        "FSIZES",

ip-10-128-42-167:bbi$ ( make && cd integration && go test -run TestBbiCompletesLocalExecution )
cd cmd/bbi; go install -ldflags "-X main.buildTime=`date +%FT%T%z`"
--- FAIL: TestBbiCompletesLocalExecution (6.41s)
    --- FAIL: TestBbiCompletesLocalExecution/acop[INT-LOCAL-001] (6.40s)
        bbi_local_test.go:32: Beginning work with /tmp/bbi-test-LRrS3w9/working as the root
        --- FAIL: TestBbiCompletesLocalExecution/acop[INT-LOCAL-001]/acop (6.38s)
            bbi_local_test.go:65:
                        Error Trace:    nonmem.go:60
                                                                bbi_local_test.go:65
                                                                wrapt.go:79
                        Error:          file "/tmp/bbi-test-LRrS3w9/working/acop/acop/FDATA.csv" exists
                        Test:           TestBbiCompletesLocalExecution/acop[INT-LOCAL-001]/acop
FAIL
exit status 1
FAIL    github.com/metrumresearchgroup/bbi/integration  6.417s
```

</details>